ArtOfIllusion/src/artofillusion/Scene.java

```diff
@@ -1159,21 +1165,6 @@ public Texture getDefaultTexture()
     return sel;
   }
 
-  /** Return true if any errors occurred while loading the scene.  The scene is still valid
-      and usable, but some objects in it were not loaded correctly. */
-
-  public boolean errorsOccurredInLoading()
-  {
-    return errorsLoading;
-  }
-
-  /** Get a description of any errors which occurred while loading the scene. */
-
-  public String getLoadingErrors()
-  {
-    return (loadingErrors == null ? "" : loadingErrors.toString());
-  }
-
```

Comment:
My one concern here is that these are public methods - just removing them could break existing scripts or plugins.

I'd rather see these two methods `@Deprecated` (with a note to use `getErrors()`) and then change the implementation to use `getErrors()` internally.

Reply:
I did not find any usages of this methods across all plugins/scripts from aoisp plugin portal. So in my opinion it is safe enough removing this.

Reply:
On further consideration, I think that this one is safe.